moveit_ros/moveit_servo/test/launch/test_servo_collision.test.py

```diff
@@ -16,10 +16,9 @@ def generate_test_description():
 
 
 class TestGTestProcessActive(unittest.TestCase):
-    def test_gtest_run_complete(
-        self, proc_info, servo_gtest, test_container, ros2_control_node
-    ):
-        proc_info.assertWaitForShutdown(servo_gtest, timeout=4000.0)
+    def test_gtest_run_complete(self, servo_gtest):
+        self.proc_info.assertWaitForShutdown(servo_gtest, timeout=4000.0)
+        time.sleep(5)
```

Comment:
5 seconds seems like a long time to sleep.  I'll need to test this to understand better what it is doing.

Reply:
I thought the same, but I cannot reproduce the flaky errors in my local. Tested out 2 and 5 seconds, was still failing at 2. Passed 6/6 with 5 seconds sleep. The problem seems to be a concurrency issue where the test result file would not be generated yet when gtest looks up for it.